### ISSUE TITLE
How do I integrate with social networks facebook and twitter?

### ISSUE THREAD

**Issue by jhonvidal** [Pyc0kw4k/djangocms_miblog#2]
would be very interesting to implement this functionality, it is very important post from the blog post to social networks, please take into account this functionality.

**Comment by Pyc0kw4k** [comment]
yes , I have already implemented this feature with Facebook, however , it is not yet fully functional I'll implement it on facebook and twitter.

**Comment by jhonvidal** [comment]
Please, it would be helpful that are published

**Comment by jhonvidal** [comment]
and comments of each post as you integrate

**Comment by jhonvidal** [comment]
Please how add comments of each post as you integrate and facebook and twitter
